src/aave-v2/SupplyVaultBase.sol

```diff
@@ -71,16 +72,26 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe {
     function __SupplyVaultBase_init_unchained(address _poolToken)
         internal
         onlyInitializing
-        returns (ERC20 underlyingToken)
+        returns (IERC20 underlyingToken)
     {
         if (_poolToken == address(0)) revert ZeroAddress();
 
         poolToken = _poolToken;
 
-        underlyingToken = ERC20(IAToken(_poolToken).UNDERLYING_ASSET_ADDRESS());
+        underlyingToken = IERC20(IAToken(_poolToken).UNDERLYING_ASSET_ADDRESS());
         underlyingToken.safeApprove(address(morpho), type(uint256).max);
     }
 
+    /// EXTERNAL ///
+
+    function transferTokens(
+        address _asset,
+        address _to,
+        uint256 _amount
+    ) external onlyOwner {
+        IERC20(_asset).safeTransfer(_to, _amount);
```

Comment:
I don't think we should allow the owner to withdraw any token. Tokens stuck on a vault should not be moved IMO, and we should only allow the owner to withdraw $MORPHO tokens

The only tangible argument is that under corruption, the owner could withdraw the underlying asset. A compromise could be to just disable the withdrawing of the underlying asset

Reply:
if compromised the DAO can upgrade and do whatever it wants. IMO the argument does really work here.

Reply:
Fair point... Still not a fan of allowing withdrawing everything but at least it's not worse than before ahah